openedx/core/djangoapps/course_groups/migrations/rerun_0006.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [ $# -ne 1 ]; then
+    echo $0: usage: rerun_0006.sh environment
+    exit 1
+fi
+
+./manage.py lms migrate --settings=$1 course_groups 0005 --fake
+./manage.py lms migrate --settings=$1 course_groups 0006
```

Comment:
Users might want to append other options too (e.g. `db-dry-run`), so why not just throw all the arguments on the end?